code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -28,6 +28,7 @@
 	var/undershirt = "Nude" //Which undershirt the player wants
 	var/socks = "Nude" //Which socks the player wants
 	var/backbag = DBACKPACK		//Which backpack type the player has chosen.
+	var/jumps	= SUIT			//Whether we're wearing a skirt to empower our coding socks

```

Comment:
Ditto variable name

Reply:
The character-defining variables listed under the preference datum share the same name as those variables set in the human_defines from most others I've seen (backbag, socks, undershirt, facial_hair_color/style, skin_tone, etc.)